vcpkg/commands/export.md

```diff
@@ -13,15 +13,49 @@ vcpkg export [options] {<package>... | --x-all-installed}
 
 ## Description
 
-Export built packages from the [installed directory](common-options.md#install-root) into a standalone developer SDK.
+The `export` command exports built packages from the [installed directory](common-options.md#install-root) into a standalone developer SDK.
 
 `export` produces a standalone, distributable SDK (Software Development Kit) that can be used on another machine without separately acquiring vcpkg. It contains:
 
 1. The prebuilt binaries for the selected packages
 2. Their transitive dependencies
 3. [Integration files](#standard-integration), such as a [CMake toolchain][cmake] or [MSBuild props/targets][msbuild]
 
-`export` must be used from [Classic Mode](../users/classic-mode.md). [Manifest Mode](../users/manifests.md) is unsupported.
+[!NOTE]
+This command's behavior is different in [Classic Mode](../users/classic-mode.md) and [Manifest Mode](../users/manifests.md)
+
+The `export` command does not install any packages or transitive dependencies. It only exports packages that are already installed. 
+
+Refer to [Manifest Mode](export.md#Manifest-Mode) or [Classic Mode](export.md#Classic-Mode) for more details.
+
+## <a name="Classic-Mode"></a>Classic Mode
+
+In classic mode, `vcpkg export` accepts [triplet-qualified package specification](install.md#package-syntax) arguments (for example: `zlib:x64-windows`)
+
+When using `vcpkg export` in classic mode, you explicitly specify the packages to export by adding `<portname:triplet>` arguments to the command line. Each argument corresponds to a specific package and its configuration (defined by the port and triplet).
+
+For example, to export the `sqlite` package for `x64-windows` and `x64-linux`, use:
+
+```no-highlight
+vcpkg export sqlite:x64-windows sqlite:x64-linux --zip
+```
+
+This command exports the specified packages in zip format. Both `sqlite:x64-windows` and `sqlite:x64-linux` must be installed prior to running `vcpkg export`.
+
+## <a name="Manifest-Mode"></a>Manifest Mode
+
+In manifest mode, the `vcpkg export` command automatically exports everything currently installed according to the specifications in the manifest file (`vcpkg.json`). This streamlined approach embodies the philosophy of manifest mode, where the entire state of installed packages is determined by the manifest file.
```

Comment:
```suggestion
In manifest mode, the `vcpkg export` command exports everything currently installed according to the dependencies declared in the manifest file (`vcpkg.json`). This approach embodies the philosophy of manifest mode, where the entire state of installed packages is determined by the manifest file.
```
 I think this needs a rewrite, since the packages in `vcpkg_installed` can actually differ from the packages declared in `vcpkg.json`.
 
> In manifest mode, the command exports all currently installed packages. The installation directory includes all the packages declared in the manifest (`vcpkg.json`) as well as their transitive dependencies. Run `vcpkg install` before using this command to ensure that all required packages are installed.